app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -399,4 +401,13 @@ public static Intent buildEmailIntent(String text) {
     emailIntent.setType("message/rfc822");
     return emailIntent;
   }
+
+  /**
+   * Marks a text input field as mandatory (appends * at end)
```

Comment:
Does this enforce the "mandatory"? Does this add the required hint?

Reply:
No it doesn't, that code is [here](https://github.com/TeamAmaze/AmazeFileManager/blob/028600a08e677f0b418b010b6596187bf62bdd59/app/src/main/java/com/amaze/filemanager/ui/dialogs/SmbConnectDialog.java#L156-L191) and [here](https://github.com/TeamAmaze/AmazeFileManager/blob/028600a08e677f0b418b010b6596187bf62bdd59/app/src/main/java/com/amaze/filemanager/ui/dialogs/SmbConnectDialog.java#L287-L306)